test/io.jl

```diff
@@ -366,10 +366,10 @@ module TestIO
 
     # Overwrites file if append == false
     writetable(tf, df1)
-    writetable(tf, df2)
-    @test readtable(tf) == df2
+    writetable(tf, df3)
+    @test readtable(tf) == df3
 
-    # Enforces matching column names iff append == true && header == true
+    # Enforces matching column names if append == true && header == true
```

Comment:
Is this change intended? "Iff" usually means "if and only if", so that's not a typo.